src/rest_vol_object.c

```diff
@@ -578,10 +746,368 @@ RV_object_specific(void *obj, const H5VL_loc_params_t *loc_params, H5VL_object_s
             break;
 
         /* H5Ovisit(_by_name) */
-        case H5VL_OBJECT_VISIT:
-            FUNC_GOTO_ERROR(H5E_OBJECT, H5E_UNSUPPORTED, FAIL,
-                            "H5Ovisit and H5Ovisit_by_name are unsupported");
+        case H5VL_OBJECT_VISIT: {
+            const char *parent_object_header = NULL;
+            iter_data   object_iter_data;
+            hsize_t     idx_p = 0;
```

Comment:
This variable can just be eliminated since there's no index pointer saving capability for H5Ovisit